APPS-2132: Refactor: Provide secure storage for sensitive PBC data (#2)
spryker/propel-encryption-behavior: README.md

> # Spryker - PropelEncryptionBehavior

Reviewer: Please add information and usage example for resetInstance()

> # Spryker - PropelEncryptionBehaviour

Reviewer 2: Typo Behavior

[![Build Status](https://github.com/spryker/propel-encryption-behavior/workflows/CI/badge.svg?branch=master)](https://github.com/spryker/propel-encryption-behavior/actions?query=workflow%3ACI+branch%3Amaster)
[![codecov](https://codecov.io/gh/spryker/propel-encryption-behavior/branch/master/graph/badge.svg?token=L1thFB9nOG)](https://codecov.io/gh/spryker/propel-encryption-behavior)
[![Latest Stable Version](https://poser.pugx.org/spryker/propel-encryption-behavior/v/stable.svg)](https://packagist.org/packages/spryker/propel-encryption-behavior)
[![Minimum PHP Version](https://img.shields.io/badge/php-%3E%3D%207.3-8892BF.svg)](https://php.net/)
[![PHPStan](https://img.shields.io/badge/PHPStan-level%208-brightgreen.svg?style=flat)](https://phpstan.org/)
[![License](https://poser.pugx.org/spryker/propel-encryption-behavior/license)](https://packagist.org/packages/spryker/propel-encryption-behavior)

Seamlessly encrypt/decrypt Propel2 data fields. This library is a *plugin* for the [Propel2 ORM framework](http://propelorm.org/).

> Also, this library was forked from [Athens\Encryption](https://github.com/AthensFramework/Encryption). 

Reviewer: ```suggestion
The library is a fork of [Athens\Encryption](https://github.com/AthensFramework/Encryption). 
```

    echo $o->getMySecretData();
    // "Some data that will be encrypted."

> // If you need to use different passphrase in different queries:

Reviewer 3: ```suggestion
    // If you need to use different passphrases in different queries:
```
